grpc.go

```diff
@@ -324,6 +324,8 @@ func (a *agentGRPC) CreateContainer(ctx context.Context, req *pb.CreateContainer
 		return emptyResp, err
 	}
 
+	ociSpec.Linux.IntelRdt = nil
```

Comment:
Shouldn't this be controlled by runtime instead?

Reply:
@sboeuf With https://github.com/containers/virtcontainers/pull/568, this should not be needed.

Reply:
@sameo Yes I know, but it doesn't, I have to investigate. I have removed the commit in the meantime.